### PR TITLE
docs: add decision record about the usage of this repository

### PR DESCRIPTION
**Pull request by paullatzelsperger**
## WHAT

Adds a decision record about how this repo is going to be used, and adds rudimentary information about the PR process. More information about this will come in a future PR.

## WHY

traceable decisions

### CODE REVIEW

**Review by jimmarino — APPROVED** [review]

**Review by wolf4ood — APPROVED** [review]

**Comment by paullatzelsperger** [comment]
\cc @maximilianschmidt_zf

**Comment by paullatzelsperger** [comment]
\cc @maximilianschmidt-zf

File: docs/decision-records/2023-07-17_usage_of_this_repository/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Usage of this repository
+
+## Decision
+
+All documentation regarding specification and standards revolving around SSI and related topics should be hosted in this
+repository. No documentation regarding any particular implementation should be hosted there. Existing documents
+referring to implementation items will be deleted.
+
+Once concrete implementations exist, they will be linked in the SSI Docu repository.
```
Comment (OSchlienz) [inline docs/decision-records/2023-07-17_usage_of_this_repository/README.md:9]: @stefan-ettl we should discuss this with Manti, he derived the Standard documents from here

Comment (paullatzelsperger) [reply docs/decision-records/2023-07-17_usage_of_this_repository/README.md:9]: @OSchlienz this PR is open for 2 days already, do you think you could reach an accord until EOB today?

Comment (OSchlienz) [reply docs/decision-records/2023-07-17_usage_of_this_repository/README.md:9]: @paullatzelsperger the architecture week is until tomorrow, not shure if I will get a response from Manti.

Comment (jimmarino) [reply docs/decision-records/2023-07-17_usage_of_this_repository/README.md:9]: @OSchlienz @paullatzelsperger Can we merge this to keep the pace and if there are issues we can always amend with a follow-up DR?

Comment (stefan-ettl) [reply docs/decision-records/2023-07-17_usage_of_this_repository/README.md:9]: Let's merge it for now please. We could apply changes afterwards if necessary.

**Pull request merged**